signalfx-org-metrics/metrics.yaml

```diff
@@ -1,3 +1,255 @@
+
+sf.org.apm.numAddSpansCallsByToken:
+  brief: The number of calls to a SignalFx ingest endpoint for a specific access token.
+  description: 'The number of calls to a SignalFx ingest endpoint for a specific access token.
+  
+  Dimension(s): `orgId`, `tokenId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numAddSpansCallsByToken
+
+sf.org.apm.numAddSpansCalls:
+  brief: The number of calls to a SignalFx ingest endpoint.
+  description: 'The number of calls to a SignalFx ingest endpoint.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numAddSpansCalls
+
+sf.org.apm.invalidSpansReceived:
+  brief: 
+  description: 
+  metric_type: counter
+  title: sf.org.apm.invalidSpansReceived
+
+sf.org.apm.invalidSpansReceivedByToken:
+  brief: 
+  description: 
+  metric_type: counter
+  title: sf.org.apm.invalidSpansReceivedByToken
+
+sf.org.apm.numTracesReceived:
+  brief: 
+  description: 
+  metric_type: counter
+  title: sf.org.apm.numTracesReceived
+
+sf.org.apm.numTroubleshootingMetricSets10s:
+  brief: 
+  description: 
+  metric_type: counter
+  title: sf.org.apm.numTroubleshootingMetricSets10s
+
+sf.org.apm.numSpansDroppedInvalid:
+  brief: The number of invalid spans SignalFx receives.
+  description: 'The number of invalid spans SignalFx receives. A span can be invalid
+    if there is no start time, trace ID, or service associated with it.
+    
+    Dimension(s): `orgId`
+    
+    Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansDroppedInvalid
+
+sf.org.apm.numSpansDroppedOversize:
+  brief: The number of spans SignalFx receives that are too large to process.
+  description: 'The number of spans SignalFx receives that are too large to process.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansDroppedOversize
+
+sf.org.apm.numSpansDroppedInvalidByToken:
+  brief: The number of invalid spans SignalFx receives for a specific access token.
+  description: 'The number of invalid spans SignalFx receives for a specific access token.
+    A span can be invalid if there is no start time, trace ID, or service associated with it.
+    
+    Dimension(s): `orgId`, `tokenId`
+    
+    Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansDroppedInvalidByToken
+
+sf.org.apm.numSpansDroppedOversizeByToken:
+  brief: The number of spans SignalFx receives that are too large to process
+    for a specific access token.
+  description: 'The number of spans SignalFx receives that are too large to process
+    for a specific access token.
+  
+  Dimension(s): `orgId`, `tokenId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansDroppedOversizeByToken
+
+sf.org.apm.grossContentBytesReceivedByToken:
+  brief: The size of bytes SignalFx receives off the wire for a specific access
```

Comment:
Wherever you have "the size of bytes" I think "the volume of bytes", or "the total amount of bytes" is more appropriate.